cfme/utils/appliance/db.py

```diff
@@ -223,6 +223,9 @@ def enable_internal(self, region=0, key_address=None, db_password=None, ssh_pass
                 self.logger.warning(
                     'Failed to set --dbdisk from the appliance. On 5.9.0.3+ it will fail.')
 
+        # make sure the dbdisk is unmounted, RHOS ephemeral disks come up mounted
+        client.run_command('umount {}'.format(db_disk))
```

Comment:
**optional** to support debugging to a debug/info log of the result/failure

Reply:
Adding this, good call @RonnyPfannschmidt